manifests/stage-nv-peer-mem-driver/0010_nv-peer-mem-driver-ds.yaml

```diff
@@ -99,3 +99,12 @@ spec:
       nodeSelector:
         feature.node.kubernetes.io/pci-15b3.present: "true"
         nvidia.com/gpu.present: "true"
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
maybe we can also add init-container with the following validation for he nvidia driver [1] 
This is how the GPU operator does it 

[1] - https://github.com/NVIDIA/gpu-operator/blob/master/assets/state-container-toolkit/0400_container_toolkit.yml#L26-L34

Reply:
Yes, I agree. Such init container can be a good addition. I updated the PR.